controllers/utils.go

```diff
@@ -31,21 +33,66 @@ func replaceProjectName(replacement string, m map[string]interface{}) {
 		case string:
 			s := m[k].(string)
 			m[k] = strings.ReplaceAll(s, "${PROJECT_NAME}", replacement)
+		case []string:
+			for i, s := range v.([]string) {
+				v.([]string)[i] = strings.ReplaceAll(s, "${PROJECT_NAME}", replacement)
+			}
 		case int64:
+		case []int64:
 		case int32:
+		case []int32:
 		case int:
+		case []int:
 		case bool:
+		case []bool:
 			continue
-		case []interface{}:
-			for _, elem := range v.([]interface{}) {
-				replaceProjectName(replacement, elem.(map[string]interface{}))
+		case []map[string]interface{}:
+			for _, a := range v.([]map[string]interface{}) {
+				replaceProjectName(replacement, a)
 			}
+		case []interface{}:
+			replaceProjectNameInSlice(replacement, m[k].([]interface{}))
 		case interface{}:
 			replaceProjectName(replacement, m[k].(map[string]interface{}))
 		}
```

Comment:
Should we add a default case to at least log an error?

Reply:
I tried to add a default case for the types that don't need replacement at all, e.g. `int`, `[]bool` etc. But the `case interface{}` is fulfilling pretty much all types. The panic happens in the cast to `m[k].(map[string]interface{})` even if `v` is an int, but an int is also an `interface{}` and thus not reaching the default case. It probably doesn't hurt though either.